### PR TITLE
bgnpcgn-urd-Arab-Latn-2007

### PR DESCRIPTION
**Pull request by AhMohsen46**
Done except for #572

### CODE REVIEW

**Comment by AhMohsen46** [comment]
blocked by #572

File: maps/bgnpcgn-urd-Arab-Latn-2007.yaml [hunk 1]
```diff
@@ -0,0 +1,456 @@
+---
+authority_id: bgnpcgn
+id: 2007
+language: iso-639-3:prs # prs stands for Dari (https://iso639-3.sil.org/code/prs&_ga=GA1.2.2054538372.1574092823)
```
Comment (ronaldtse) [inline maps/bgnpcgn-urd-Arab-Latn-2007.yaml]: Change to `iso-639-2:urd`?

File: maps/bgnpcgn-urd-Arab-Latn-2007.yaml [hunk 2]
```diff
@@ -0,0 +1,456 @@
...
+  - 4. Note that the short vowels in the Urdu examples are not
+    pointed.
+  - 5. Occasionally, sequences of /z/ or /s/ plus /h/ may be
+    encountered, i.e. z·h, s·h. These may be romanized with the
+    Unicode
+    'center dot' (U+00B7) separating the two letters, to
+    distinguish them from the digraphs /zh/ and /sh/.
+
+tests:
+  # - source: بوغدِی
```
Comment (ronaldtse) [inline maps/bgnpcgn-urd-Arab-Latn-2007.yaml:58]: Are these working, or to be done?

**Review by ronaldtse — CHANGES_REQUESTED** [review]
@AhMohsen46 could you have a look at the comments? Thanks!

File: maps/bgnpcgn-urd-Arab-Latn-2007.yaml (hunk 1 above)
Comment (AhMohsen46) [reply maps/bgnpcgn-urd-Arab-Latn-2007.yaml]: yes sure, will do this immediately

File: maps/bgnpcgn-urd-Arab-Latn-2007.yaml (hunk 2 above)
Comment (AhMohsen46) [reply maps/bgnpcgn-urd-Arab-Latn-2007.yaml:58]: so, the commented out examples, are from the docs, however, they are dependent on the cases in the blocking issue, as soon as it's resolved, the examples will be uncommented and should be okay to run without failing

Comment (ronaldtse) [reply maps/bgnpcgn-urd-Arab-Latn-2007.yaml:58]: In this case can you add the link to the blocking issue inside the comment so people reading the map can know this? Thanks!

Comment (AhMohsen46) [reply maps/bgnpcgn-urd-Arab-Latn-2007.yaml:58]: yes, this makes perfect sense, will do it immediately

Comment (AhMohsen46) [reply maps/bgnpcgn-urd-Arab-Latn-2007.yaml:58]: done
added the following note
  - Commented tests are blocked by this issue https://github.com/interscript/interscript/issues/572
    depends on the different ways of handling ي to y or e AND و to u or o

**Comment by ronaldtse** [comment]
@AhMohsen46 the specs should be working now, could you help rebase? Thanks!

**Comment by AhMohsen46** [comment]
@ronaldtse rebased, pushed
all good to go except for #572

**Comment by ronaldtse** [comment]
Thanks @AhMohsen46 !

**Pull request merged**